docs/references/chrome-extension/overview.mdx

```diff
@@ -27,24 +29,26 @@ The authentication options for a Chrome Extension will differ depending on how t
 | Web3 | | | ✅ |
 
 > [!WARNING]
-> Our Chrome Extension SDK currently does not fully support Sync Host on side panels. The side panel would need to be closed and then reopened to update the authentication status
+> Our Chrome Extension SDK currently does not fully support Sync Host on side panels. Currently, if a user authenticates in your web app, they need to close and reopen the side panel to update their auth status.
 
-## Chrome Extension CRX ID
+## Configure a consistent CRX ID
```

Comment:
I think we should change the order of these? This was initially established when the CRX ID guide was required. Maybe the following order?

1. React Router 
2. Sync Host
3. `createClerkClient()`
4. Deploy to production
5. Configure a CRX ID

Reply:
can do

Reply:
updated! [b089102](https://github.com/clerk/clerk-docs/pull/1004/commits/b089102b7f14df60439d79fd7c6313e1a735050a)